.gitreview

```diff
@@ -2,4 +2,4 @@
 host=review.opendev.org
 port=29418
 project=openstack/kayobe-config.git
-defaultbranch=stable/yoga
```

Comment:
This is going to break a few of our workflows. The host image build ones are fine because they're broke at the moment anyway but container image builds and CI cleanup are both affected
https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/yoga/.github/workflows/stackhpc-container-image-build.yml#L68
https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/yoga/.github/workflows/stackhpc-ci-cleanup.yml#L33

Reply:
Good catch.